JustTrack/EventsGenerator/EventListTemplate.jet

```diff
@@ -2,7 +2,7 @@
 
 /*example
 
-@objcMembers public class JEEventExample: NSObject, JEEvent {
+public class EventExample: NSObject, Event {
```

Comment:
Do we still need to inherit from NSObject?

Reply:
Let me research this.

Reply:
Done